.github/workflows/mixin.yml

```
@@ -14,7 +14,7 @@ jobs:
       - name: install Go
         uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32 # v5.0.2
         with:
-          go-version: 1.23.x
+          go-version: 1.24.x
```

```suggestion
          go-version-file: 'go.mod'
```

Thats what I favorite at least. It uses the go directive from go.mod

---

But won't that make it use 1.23? I don't think that's a good idea.